docs/deployment.md

```diff
@@ -0,0 +1,90 @@
+# Deployments
+
+## Next.js app
+
+The Next.js app can be deployed as a static HTML export or as a Node.js server, depending on your needs. The [Next.js deployment documentation](https://nextjs.org/docs/deployment) provides more information on how to deploy your application.
+
+## Storybook
+
+Storybook can be deployed as a static HTML export. The [Storybook deployment documentation](https://storybook.js.org/docs/react/sharing/publish-storybook) provides more information on deployment options.
+
+### GitHub Pages
+
+[GitHub Pages](https://pages.github.com/) may be a good option for hosting Storybook if you only need one environment to represent the latest state of your main branch.
```

Comment:
small nit to reverse the order to something like
```
# Set up Storybook deployment
this guide helps set up deploying storybook to github pages on every merge to main

... steps ...

## alternatives
since storybook can be deployed as a static html export you can also deploy to S3 (see storybook for more info)

alternatively you can also use chromatic which is a separate service (link)
```